src/Swift.Bindings/src/Model/TypeDecl/MethodDecl.cs

```diff
@@ -29,10 +28,20 @@ public sealed record MethodDecl : BaseDecl
         /// </summary>
         public required List<ArgumentDecl> CSSignature { get; set; }
 
-		/// <summary>
-		/// Indicates if method can throw an exception.
-		/// </summary>
-		public required bool Throws { get; set; }
+        /// <summary>
+        /// Indicates if method can throw an exception.
+        /// </summary>
+        public required bool Throws { get; set; }
+
+        /// <summary>
+        /// Generic parameters of the method.
+        /// </summary>
+        public required List<string> GenericParameters { get; set; }
```

Comment:
Is `GenericParameters` redundant with `CSSignature`?

Reply:
`GenericParameters` is the list of generic type params in the order they appear in the function declaration, `Signature` contains a list of arguments (e.g. one generic type can appear multiple times in function signature)